src/components/Global/ThemeButton.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="flex justify-end items-center space-x-2 mx-1">
+    <span class="text-sm text-gray-800">Light</span>
+    <div>
+      <input
+        @click="toggleDarkMode"
+        type="checkbox"
+        name=""
+        id="toggle"
+        v-model="isDarkMode"
+        class="hidden"
+      />
+      <label for="toggle">
+        <div
+          class="
+            w-7
+            h-4
+            sm:w-9 sm:h-5
+            flex
+            items-center
+            bg-gray-300
+            rounded-full
+            sm:p-1
+          "
+        >
+          <div
+            class="
+              toggle-dot
+              w-3.5
+              h-3.5
+              sm:w-4 sm:h-4
+              bg-white
+              rounded-full
+              shadow-md
+              transform
+              duration-300
+              ease-in-out
+            "
+          ></div>
+        </div>
+      </label>
+    </div>
+    <span class="text-sm text-gray-400">Dark</span>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      isDarkMode: false,
+    };
+  },
+  methods: {
+    toggleDarkMode() {
+      // var checkbox = document.getElementById('#toggle')
+      if (this.isDarkMode) {
+        localStorage.removeItem("theme");
```

Comment:
How about we also implement a media query for prefers-color-scheme

Reply:
prefers-color-scheme is used in mounted().
When page is loaded at the beginning